### PR TITLE
Added script for scraping dark websites

### PR DESCRIPTION
**Pull request by SreeHaran**
**script name** - dark_web_scraper.py

**What have you Changed** 

Added new folder named **scrap_dark_websites**

### Self Check(Tick After Making pull Request)

- [x] This issue was assigned to me.
- [x] One Change in one Pull Request
- [x] My file is in the proper folder
- [x] I am following clean code and Documentation and my code is well-linted with flake8.
- [x] I have added README.md with my script

If the issue was not assigned to you Please don't make a PR. It will mark as invalid.

Join Us on Discord:- https://discord.gg/JfbK3bS

## acknowledgment
- [x] I have tested the script and distributed it under an MIT license.

### CODE REVIEW

File: scrap_dark_websites/README.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# DARK WEB SCRAPER
+
+This script allows to scrap the deep and dark websites (.onion URLs) anonymously with Tor proxy.
+
+## Requirements
+
+Tor Browser is required for the proxy connection. Other than no python libraries are required, all the libraries used in this are built-in python packages.
```
Comment (codeperfectplus) [inline scrap_dark_websites/README.md]: requests is a 3rd party library. it's not built-in.

**Review by codeperfectplus — CHANGES_REQUESTED** [review]
Please update the documentation & add requirements.txt file .

**Comment by SreeHaran** [comment]
@codePerfectPlus Added Requirements.txt and Updated the README.md. Please Check!

**Review by codeperfectplus — APPROVED** [review]
Thanks for making your contributions.

you can join us on telegram for more updates about Hacktoberfest.
Hacktoberfest Channel:- t.me/hacktober2022

Happy hacktoberfest

**Pull request merged**